lighthouse-core/lib/traces/devtools-timeline-model.js

```diff
@@ -36,11 +36,14 @@ class TimelineModel {
     // populate with events
     this._tracingModel.reset();
 
-    ConsoleQuieter.mute({prefix: 'timelineModel'});
-    this._tracingModel.addEvents(events);
-    this._tracingModel.tracingComplete();
-    this._timelineModel.setEvents(this._tracingModel);
-    ConsoleQuieter.unmuteAndFlush();
+    try {
+      ConsoleQuieter.mute({prefix: 'timelineModel'});
```

Comment:
what does this do?

Reply:
> what does this do?

any console messages that are absorbed are later logged with that as the logger prefix/title

Reply:
oh I didn't mean this particular line, just why is this try/finally necessary now?

Reply:
Previous to this patch landing, when you run `mocha lighthouse-core/test/lib/traces/devtools-timeline-model-test.js` it just exits silently. That's because console-quieter was also muting the logs from mocha. They never got flushed (the `finally`) because of an exception.

Reply:
oh gotchaaaa, mocha really just calls `console.log`? I see

Reply:
yah it does. found that this morning. was kinda surprised they aren't doing `process.std*.write()`, but yeah. :)